ui/src/testing/stubs/leo-proxy-api-stub.ts

```diff
@@ -1,12 +1,8 @@
 import { ProxyApi } from 'notebooks-generated/fetch';
 
-import { stubNotImplementedError } from 'testing/stubs/stub-utils';
-
 export class LeoProxyApiStub extends ProxyApi {
   constructor() {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.